sormas-app/app/src/main/java/de/symeda/sormas/app/backend/sample/PathogenTestDtoHelper.java

```diff
@@ -64,6 +70,12 @@ protected void fillInnerFromDto(PathogenTest target, PathogenTestDto source) {
         target.setTestType(source.getTestType());
         target.setTestedDisease(source.getTestedDisease());
         target.setTestedDiseaseDetails(source.getTestedDiseaseDetails());
+
+        //newly added
```

Comment:
Please remove all comments that don't provide some essential information; particularly these "newly added" comments